src/misc/programs.cpp

```diff
@@ -400,19 +400,14 @@ void CONFIG::HandleHelpCommand(const std::vector<std::string>& pvars_in)
 	}
 
 	Section_prop* psec = dynamic_cast<Section_prop*>(sec);
-	if (psec == nullptr) {
-		// Failed; maybe it's the autoexec section?
-		Section_line* pline = dynamic_cast<Section_line*>(sec);
-		if (pline == nullptr) {
-			E_Exit("Section dynamic cast failed.");
```

Comment:
:+1: for getting rid of this.  No reason to kill the entire emulator because a command line program failed.

Reply:
Yeah, luckily many (but not all) of these hard exits are unreachable. I could not trigger this one, but yeah, using `E_Exit` should be extremely rare. I remove them on sight.